components/NavLink.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import Link from 'next/link'
+
+type NavLinkProps = {
+  path: string
+  activePath?: string
+  external?: true
+  as?: string
+  className?: string
+}
+
+const NavLink: React.FC<NavLinkProps> = ({
+  children,
+  path,
+  activePath,
+  as,
+  external,
+  className
+}) => {
+  const active = path === activePath
+  const hasClass = typeof className !== 'undefined'
```

Comment:
```
if (path === activePath) {
  className += ' active'
}
```

Reply:
I need the active variable for another line as well.

Reply:
```
 <a className={`${className} ${active ? 'active' : ''}`}>
   {active && <span className="sr-only">(current)</span>}
   {children}
</a>
```